html/pages/front/default.php

```diff
@@ -116,7 +116,7 @@ function generate_front_box($frontbox_class, $content)
 }
 
 // Device rebooted boxes
-if (filter_var($config['uptime_warning'], FILTER_VALIDATE_FLOAT) !== false && $config['uptime_warning'] > 0 && $config['os'][$os]['bad_uptime'] !== true) {
+if ((filter_var($config['uptime_warning'], FILTER_VALIDATE_FLOAT) !== false && $config['uptime_warning'] > 0) && $config['os'][$os]['bad_uptime'] !== true) {
```

Comment:
my bad :) nothing at all  ...
push an updated code, i think that the issue is $config[''os"][$os]["bad_uptime"] is not available in that script so it will return always that bad_uptime !== true